Lib/Panel/HistoryPanel.php

```diff
@@ -67,8 +67,8 @@ public function beforeRender(Controller $controller) {
 				$historyStates[] = array(
 					'title' => $title,
 					'url' => array_merge($prefix, array(
-						'plugin' => 'debug_kit',
-						'controller' => 'toolbar_access',
+						'plugin' => 'DebugKit',
+						'controller' => 'ToolbarAccess',
```

Comment:
Underscore values generally work find in 2.x why does this need to change?

Reply:
In SQL log when click on the button, does not work for me.

![image](https://user-images.githubusercontent.com/42739735/48283984-2c6f8480-e42c-11e8-904c-8599beea5f2f.png)

![image](https://user-images.githubusercontent.com/42739735/48284049-54f77e80-e42c-11e8-83b6-2309614dc075.png)